test/testcafe/spec/DeviceChallengePollViewFailure_spec.js

```diff
@@ -36,7 +36,6 @@ async function setup(t) {
 test('probing and polling APIs are sent and responded', async t => {
   const deviceChallengePollPageObject = await setup(t);
   await t.expect(deviceChallengePollPageObject.getHeader()).eql('Verifying your identity');
-  await t.expect(deviceChallengePollPageObject.getSpinner().getStyleProperty('display')).eql('block');
```

Comment:
Now when we get probing response, log in happens immediately, so no spinner is show.